src/NextGen/DonationForm/ViewModels/DonationFormViewModel.php

```diff
@@ -13,76 +15,124 @@
 class DonationFormViewModel
 {
     /**
-     * @var DonationForm
+     * @var int
      */
-    private $donationForm;
+    private $donationFormId;
     /**
      * @var BlockCollection
      */
-    private $formBlockOverrides;
+    private $formBlocks;
     /**
      * @var array
```

Comment:
Let's use the `array{designId: int, ...}` syntax here to give us some help.

Reply:
FYI the `$formSettings` property will eventually be a specific type.  Soon we're going to update `$donationForm->settings` to be a `FormSettings` property type instead of a basic array.  I added _some_ array items to the doc with a note here 9bbb4973